### ISSUE TITLE
Adding the basic DA component to the merged unified community regional workflow and EMC regional workflow

### ISSUE THREAD

**Issue by TingLei-NOAA** [NOAA-EMC/regional_workflow#202]
Starting from Ratko's fork of the unified community workflow with his developments then,  the basic DA component ( without cycling of  FV3SAR ensemble using EnKF) has been added to unified regional workflow on the "low level". By the low level, it means the developments are directly started from on the -"automatic-generated" experiment-specific configuration files. This strategy also helps avoid any significant overlapping with progressing developers' work on the whole system.  

The branch is DA-merged-unfiied-EMC-regional-workflow.
https://github.com/TingLei-NOAA/regional_workflow/tree/DA-merged-unfiied-EMC-regional-workflow . 
After downloading it, in the root directory ( regional_workflow/) , 
the dir: expt_dirs/  should be moved to upper level .

DA cycling experiment using this branch has been run on Hera for "NCO real time mode" and passed the sanity test, though further evaluation is needed. 
The current goal is for developers working on the whole system (especially on high level process like controlling the experiment setup process) to review this branch and give further comments on  any consolidation of those "low level" devleopements needed for them to fit into the current unified regional workflow merged with the EMC regional workflow. 
More documentation may be added in the following notes.

**Comment by christinaholtNOAA** [comment]
@TingLei-NOAA Just to make sure I'm looking through the right parts of this, could you give a very brief overview of where I might start if I was going to run a DA workflow? Which files to edit/run would be sufficient. 

If you have this info outlined somewhere else, a link/email to that would also be great.

Thanks!

**Comment by hu5970** [comment]
Ting, thanks for adding all those da components.

The J-job and GSI analysis scripts are two key parts need to be added. They look good but need some clean-up and generalization for hourly cycling system like RAP.

We should find sometime to discuss some details, especially how to make the community and NCO scripts work together.

**Comment by TingLei-NOAA** [comment]
Ming,
Sure.  that will be great for us to have some discussion about more
details.
Regards,
Ting

______________________________

Ting Lei

IMSG at NOAA/NWS/NCEP/EMC

5830 University Research Ct., Cubicle 2765

College Park, MD 20740

Ting.Lei@noaa.gov <Joe.Smith@noaa.gov>

301-683-3624

**Comment by jwolff-ncar** [comment]
What is the status of this issue? @TingLei-NOAA @hu5970 @christinaholtNOAA

**Comment by JeffBeck-NOAA** [comment]
@jwolff-ncar, I think we're waiting on the creation of the authoritative RRFS branch in regional_workflow, then we will work on moving the DA capability into the develop branch piece by piece.

**Comment by TingLei-daprediction** [comment]
Hi, Jeff, 
I think Ming Hu and Shun Liu are working on a DA workflow for the community use.  I will be happy to help in that effort. 
Is that better to open a new issue for them and let this issue be closed?

**Comment by JeffBeck-NOAA** [comment]
@TingLei-daprediction, we can certainly do that if you prefer.  I'll go ahead and open it now and close this one.
